src/openai.ts

```diff
@@ -52,4 +52,13 @@ export const GPT_MODEL_LIST: {[key: string]: GPTModelConfig} = {
             output: 0.03,
         },
     },
+    'gpt-4o': {
+        model: 'gpt-4o',
+        maxTokens: 128000,
+        interface: 'chat',
+        cost: {
+            input: 0.01,
```

Comment:
It looks like the pricing here differs from the openAI website. Can you double-check, please?
![image](https://github.com/user-attachments/assets/d4f3de5f-4bf4-4065-9fd5-c987f2cc4a91)